src/main/java/com/wanderersoftherift/wotr/WanderersOfTheRift.java

```diff
@@ -138,5 +143,15 @@ public static void onClientSetup(FMLClientSetupEvent event) {
         private static void registerScreens(RegisterMenuScreensEvent event) {
             event.register(ModMenuTypes.RUNE_ANVIL_MENU.get(), RuneAnvilScreen::new);
         }
+
+        @SubscribeEvent
+        public static void registerEntityRenderers(EntityRenderersEvent.RegisterRenderers event) {
```

Comment:
Some of the other branches introduced a ClientRegistryEvents class. This should potentially move there. See Rune_anvil

Reply:
Done, moved to `com.wanderersoftherift/wotr/init/client/ClientRegistryEvents`